examples/provider/bid/conversant/conversant-ex1.html

```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Conversant - PubFood Example page</title>
+  </head>
+  <body>
+    <script src="../../../../build/pubfood.js"></script>
+    <script  type="text/javascript">
+
+      //add the following lines to 'Code Snippet' box of DFP third party creative within a <script> tag
+      //gpt would then call renderAd below to render the ad
+
+     /*
+      var w = window;
+      for (i = 0; i < 10; i++) {
+        w = w.parent;
+        try {
+          if (w.pubfood) {
+            w.pubfood.renderAd(document, '%%PATTERN:cn728x90adId%%');
+          }
+          break;
+        } catch(e) {continue;}
+      }
+     */
+
+      //rudimentary html validation, use stricter validation if needed
+      var validateHTML = function (snippet) {
```

Comment:
I think the better approach for `renderAd` will be to wrap the update to the DOM in an exception handler and take the `bid.adm` value at "face value". If we need logic to take different action on the creative text representation then we can go about parsing the creative/markup.

Secondly, `String.prototype.endsWith()` is not well supported yet. In general a RegExp would be preferred.